auth/login/login.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="shortcut icon"
+      href="../../assets/favicon/favicon.ico"
+      type="image/x-icon"
+    />
+    <link
+      rel="stylesheet"
+      href="https://crimsonui-beta.netlify.app/css/components.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700;800&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="../../css/main.css" />
+    <link rel="stylesheet" href="./login.css" />
+    <title>3AM TALES</title>
+  </head>
+  <body>
+    <nav class="navbar" id="main-navbar">
+      <div class="left-navbar">
+        <!-- <img src="./assets/logo/3am tales.png" alt="" class="logo-img" /> -->
+        <div class="logo-title">
+          <a href="" id="logo-title-link">3AM TALES</a>
+        </div>
+      </div>
+      <div class="center-navbar">
+        <i class="fa-solid fa-magnifying-glass"></i>
+        <input type="text" name="" id="search-bar" placeholder="Search" />
+      </div>
+      <div class="right-navbar">
+        <a href=""> <i class="fa-regular fa-heart fa-lg navbar-icons"></i></a>
+
+        <a href="" class="navbar-cart"
+          ><i class="fa-solid fa-cart-shopping fa-lg navbar-icons"></i>
+          <span>Cart</span>
+        </a>
+        <button class="btn btn-link">
+          <a href="#">Login</a>
+        </button>
+      </div>
+    </nav>
+
+    <div class="main-body">
```

Comment:
you could have used a `main` tag here instead of div. This way you will be using more semantic elements in your app

Reply:
Sure! I will correct it